app/src/main/java/app/insti/fragment/AboutFragment.java

```diff
@@ -77,7 +77,7 @@ public void onStart() {
             }}, context));
             add(new AboutCategory("Ideation", new ArrayList<AboutIndividual>() {{
                 add(new AboutIndividual("nihal111", "Nihal Singh", "nihal.jpg"));
-                add(new AboutIndividual("ydidwania", "Yashwarshan Didwania", "ydidwania.jpg"));
+                add(new AboutIndividual("ydidwania", "Yashwardhan Didwania", "ydidwania.jpg"));
```

Comment:
its "Yashvardhan Didwania". Please update this :smile: 

(src: https://github.com/ydidwania)